lib/src/screens/recipe_screen.dart

```diff
@@ -159,14 +153,45 @@ class _RecipeScreenBodyState extends State<RecipeScreenBody> {
 
     final bottom = SliverList(
       delegate: SliverChildListDelegate.fixed([
-        if (recipe.tool.isNotEmpty) ToolList(recipe: recipe),
         if (MediaQuery.of(context).size.width > 600)
-          Row(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            children: list.map((e) => Expanded(flex: 5, child: e)).toList(),
-          )
+          ...[
+            Row(
+              crossAxisAlignment: CrossAxisAlignment.start,
+              children: [
+                if (recipe.tool.isNotEmpty)
+                  Expanded(
+                    flex: 5,
```

Comment:
The flex factor is the same on all elements in this row.

It shouldn't be needed or am I wrong?

Reply:
You are right, they are not necessary. I took them out.